tasks/main.yml

```diff
@@ -24,6 +24,48 @@
   when:
     - network_state is defined
     - ansible_distribution_major_version | int < 8
+
+- name: Check if updates for network packages are available through the DNF
+    package manager due to wireless or team interfaces
+  dnf:
+    update_cache: true
+    name: "{{ network_packages }}"
+    state: latest  # noqa package-latest
+  register: dnf_package_update_info
+  check_mode: true
+  when:
+    - ansible_distribution == 'Fedora' or
+      ansible_distribution_major_version | int > 7
+    - __network_wireless_connections_defined
+      or __network_team_connections_defined
+
+- name: Check if updates for network packages are available through the YUM
+    package manager due to wireless or team interfaces
+  yum:
```

Comment:
Can https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html be used instead of having two tasks for dnf/yum? What's the reason that `update_cache` is enabled?

Reply:
https://github.com/linux-system-roles/network/pull/675#discussion_r1497609081, as suggested by Fernando, I want to check if any network package can be updated, and this is the reason that `update_cache` is used here. 

The package module can not check if a list of packages can be updated.

Reply:
I updated the commit message.